soa/README.md

```
@@ -0,0 +1,96 @@
+|---|---|
+| `supports` | Evidence or argument for |
+| `contradicts` | Evidence or argument against |
+| `similar` | Related but not identical |
+| `same` | Equivalent propositions |
+| `requires` | Dependency (B requires A) |
+| `enables` | Capability relationship (A enables B) |
+| `parent` / `child` | Tree composition |
+
+## Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)